templates/default/password_toggle.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
```

Comment:
Please rewrite this according to https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/js-bundling.md

this results in a small module which then can be used with an init-method (see comment above) like PasswordToggle.init().

I think that we should increasingly centralize JS in the plugin and gradually bring it into this form. we can therefore start directly with this JS